src/device/cuda/cooperative_groups.jl

```diff
@@ -1,42 +1,71 @@
 # C. Cooperative Groups
 
-export this_grid, sync_grid
+export this_grid, sync_grid,
+       this_multi_grid, sync_multi_grid
 
-"""
-    this_grid()
+@enum cudaCGScope::Cuint begin
+      cudaCGScopeInvalid   = 0  # Invalid cooperative group scope
```

Comment:
drop the redundant `cuda` on the values?

Reply:
Sure. (didn't learn from the CUDAapi PR 😂)